modules/mtc-deploying-a-vm.adoc

```diff
@@ -0,0 +1,77 @@
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtc-deploying-a-vm_{context}"]
+= Deploying a Virtual Machine
+
+When you have installed KubeVirt and CDI, and they are active, create a namespace.
+
+For this example we use the namespace `mig-vm`. And create the following YAML to create a Fedora Virtual Machine.
+
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-fedora
+  name: vm-fedora
+  namespace: mig-vm
+spec:
+  dataVolumeTemplates:
+  - metadata:
+      name: fedora-40-dv-template
+      namespace: mig-vm
+    spec:
+      storage:
+        resources:
+          requests:
+            storage: 8Gi
+      source:
+        http:
+          url: "https://download.fedoraproject.org/pub/fedora/linux/releases/40/Cloud/x86_64/images/Fedora-Cloud-Base-AmazonEC2.x86_64-40-1.14.raw.xz"
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-fedora
+    spec:
+      networks:
+      - name: default
+        pod: {}
+      domain:
+        ioThreadsPolicy: auto
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: datavolumedisk1
+          - disk:
+             bus: virtio
+            name: cloudinitdisk
+          interfaces:
+          - masquerade: {}
+            name: default
+        resources:
+          requests:
+            memory: 1024Mi
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - dataVolume:
+          name: fedora-40-dv-template
+        name: datavolumedisk1
+      - cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            password: fedora
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+----
+
```

Comment:
Suggestions:

- Use present tense.
-  "... and a datavolume containing the Fedora operating system, at the time of this writing, Fedora 40." OR "...operating system (at the time of this writing, Fedora 40). Here, I think a parenthetical comment might actually be better, but that's a matter of taste.